M2/Macaulay2/m2/document.m2

```diff
@@ -204,7 +204,10 @@ makeDocumentTag String      := opts -> key -> (
 -- TODO: can this be modified to fix the tag in-place? then we would only need to
 -- fix the tag in (validate, TO), rather than also in (info, TO) and (html, TO).
 fixup DocumentTag := DocumentTag => tag -> (
-    if (rawdoc := fetchAnyRawDocumentation tag) =!= null then rawdoc.DocumentTag else tag)
+    tag' := if (rawdoc := fetchAnyRawDocumentation tag) =!= null then rawdoc.DocumentTag else tag;
+    if package tag === package tag' then tag' else printerr("warning: ambiguous reference ",
```

Comment:
Is there a reason for the `then` clause here?  Should this just be `if package tag =!= package tag' then printerr(...)`?

Reply:
oh absolutely not, good catch!